test/store.spec.js

```diff
@@ -0,0 +1,255 @@
+import Userfront from "../src/index.js";
+import {
+  getCookieOptionsByName,
+  getCookieOptionsByTokenType,
+  setCookieOptionsByName,
+  setCookieOptionsByTokenType,
+  TokenCookieStoredValue,
+  ComputedValue,
+  ReadonlyTokenDerivedValue,
+  store,
+  _resetStore,
+  clearAllCookieOptions
+} from "../src/store.js";
+import {
+  removeTokenCookies,
+  setCookies,
+  createAccessToken,
+  createIdToken,
+  createRefreshToken,
+  idTokenUserDefaults,
+  defaultIdTokenProperties
+} from "./config/utils.js";
+import Cookies from "js-cookie";
+
+const tenantId = "store-spec-tenant";
+
+describe("/store.js", () => {
+  beforeEach(() => {
+    removeTokenCookies(tenantId);
+    _resetStore();
+    clearAllCookieOptions();
+    Userfront.init(tenantId);
+  })
+  describe("cookie helpers", () => {
+    it("should set and get cookie options by name", () => {
+      const options = {
+        secure: true,
+        sameSite: "None",
+        expires: 1
+      };
+
+      setCookieOptionsByName("test-cookie-1", options);
+      const retrieved = getCookieOptionsByName("test-cookie-1");
+
+      expect(retrieved).toEqual(options);
+    })
+    it("should set and get cookie options by token type", () => {
+      const options = {
+        secure: false,
+        sameSite: "Strict",
+        expires: 2
+      }
+
+      setCookieOptionsByTokenType("access", options);
+      const retrieved = getCookieOptionsByTokenType("access");
+
+      expect(retrieved).toEqual(options);
+    })
+    it("should get default cookie options for access and id tokens", () => {
+      const expected = {
+        secure: true,
+        sameSite: "Lax"
+      }
+
+      const accessOptions = getCookieOptionsByTokenType("access");
+      const idOptions = getCookieOptionsByTokenType("id")
+
+      expect(accessOptions).toEqual(expected);
+      expect(idOptions).toEqual(expected);
+    })
+    it("should get default cookie options for refresh tokens", () => {
+      const expected = {
+        secure: true,
+        sameSite: "Strict"
+      }
+
+      const refreshOptions = getCookieOptionsByTokenType("refresh");
+
+      expect(refreshOptions).toEqual(expected);
+    })
+  })
+  describe("store - tokens", () => {
+    it("should get a token value from an existing cookie", () => {
+      const accessToken = createAccessToken();
+      const idToken = createIdToken();
+      const refreshToken = createRefreshToken();
+      setCookies([
+        {
+          name: `access.${tenantId}`,
+          value: accessToken,
+          options: {}
+        },
+        {
+          name: `id.${tenantId}`,
+          value: idToken,
+          options: {}
+        },
+        {
+          name: `refresh.${tenantId}`,
+          value: refreshToken,
+          options: {}
+        }
+      ]);
+
+      expect(store.tokens.accessToken).toEqual(accessToken);
+      expect(store.tokens.idToken).toEqual(idToken);
+      expect(store.tokens.refreshToken).toEqual(refreshToken);
+    });
```

Comment:
If I replace `setCookies` with setting the cookie directly, this test fails. E.g.

```js
it.only("should get a token value from an existing cookie", () => {
  const accessToken = createAccessToken();
  Cookies.set({
    name: `access.${tenantId}`,
    value: accessToken,
    options: {},
  });
  expect(store.tokens.accessToken).toEqual(accessToken);
});
```

I can't discern why this is the case. Is `setCookies` mocked in some way?